sdcm/nemesis.py

```diff
@@ -681,7 +685,8 @@ def modify_table_default_time_to_live(self):
             is exceeded, Cassandra tombstones the table.
             default: default_time_to_live = 0
         """
-        self._modify_table_property(name="default_time_to_live", val=random.randint(864000, 630720000))  # max allowed TTL - 20 years (630720000)
+        self._modify_table_property(name="default_time_to_live", val=random.randint(864000, 630720000),
+                                    test_counters_table=False)  # max allowed TTL - 20 years (630720000)
```

Comment:
please add a comment or explanation in the commit message of what is the issue that required this.